source/adressering-og-varsler.rst

```diff
@@ -0,0 +1,254 @@
+
+        Logg deg inn på *signering.posten.no/logginn* for å signere dokumentet.
+
+        Rekker du ikke å signere innen fristen? Usignerte dokumenter slettes når     fristen går ut. Kontakt *avsender* for å få dokumentet tilsendt på nytt.
+
+        Hilsen Posten
+
+
+..  tabs::
+
+    ..  tab:: SMS 1. varsel
+
+        Du har dokument til signering fra *Avsender*. Logg inn og signer på *signering.posten.no/logginn* innen *signeringsfristen*.
+         
+    ..  tab:: SMS 2./3. varsel
+
+        Du har et usignert dokument fra *Avsender*. Logg inn og signer på *signering.posten.no/logginn* innen *signeringsfristen*.
+         
+
+Varsel om dokument til signering, ved adressering uten fødselsnummer
+____________________________________________________________________
+
+..  tabs::
+         
+    ..  tab:: E-post 1. varsel
+
+        Hei!
+        Du har fått en forespørsel om å signere et dokument fra *Avsender*: *Dokumenttittel*.
+        
+        [Dokumentet er nå signert av #antall og må signeres innen / Dokumentet må signeres innen] *signeringsfrist*.
+        
+        Du kan signere med *disse elektroniske ID-ene*.
+        
+        Slik signerer du:
+        1) Klikk på lenken under
+        2) Skriv inn sikkerhetskode XXXX
+        3) Les og signer dokumentet
+        
+        https://signering.posten.no/uniklenke
+        
+        Hilsen Posten
+         
+    ..  tab:: E-post 2. varsel
+
+        Hei!
+        Vi vil minne om at du fortsatt har et dokument til signering fra *Avsender*: *Dokumenttittel*.
+        
+        [Dokumentet er nå signert av #antall og må signeres innen / Dokumentet må signeres innen] *signeringsfrist*.
+               
+        Du kan signere med *disse elektroniske ID-ene*.
+        
+        Slik signerer du:
+        1) Klikk på lenken under
+        2) Skriv inn sikkerhetskode XXXX
+        3) Les og signer dokumentet
+        
+        https://signering.posten.no/uniklenke
+        
+        Rekker du ikke å signere innen fristen?
+        Usignerte dokumenter slettes når fristen går ut. Kontakt *Avsender* fpr å få dokumentet tilsendt på nytt.
+               
+        Hilsen Posten
+
+.. tabs::
+         
+    ..  tab:: SMS 1. varsel
+
+        Hei! *Avsender* ber deg signere et dokument. Bruk kode XXXX på https://signering.posten.no/uniklenke før *signeringsfristen*.
+         
+    ..  tab:: SMS 2./3. varsel
+
+        Hei! Husk signering for *Avsender*. Bruk kode XXXX på https://signering.posten.no/uniklenke før *signeringsfristen*.
+
+
+
+Etter signering: Varsel om oppsalg til digital postkasse
+________________________________________________________
+
+Etter at en undertegner har signert et dokument, vil den i *disse tilfeller* få mulighet til å opprette en digital postkasse. Hvis avsender er privat, vil undertegner få mulighet til å opprette konto hos Digipost, og hvis avsender er offentlig vil undertegner kunne velge digital postkasse på Norge.no.
```

Comment:
Bruke et annet ord for undertegner enn "den"? "Vil den i disse tilfeller...". Bruke "vedkommende" eller "man"?